# Robot's current position
float64 x_pos
float64 y_pos
# Robot's current velocity
float64 x_vel
float64 y_vel